app/views/historic_appointments/_role_appointments_list.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="block historic-people-list">
-  <div class="inner-block">
+  <div class="inner-block govuk-!-padding-0">
```

Comment:
We try not to use these override classes where possible - they use `!important`, so can quickly lead to CSS-soup.

Reply:
@injms thanks - any suggestions on alternatives?

Reply:
Unfortunately it might be rebuilding that layout using GOVUK Frontend's `govuk-grid-` or creating a new CSS class in the app. I'd go for using Frontend if it were me - more short term work - but work that we'd do anyway - and clearer in the longer term.

Reply:
Hmm, after our conversation I think this opens a much bigger rebuild of the whole page. In this case I think it's acceptable to use the override class, maybe with a `TODO` comment near it, since it should be compontenised in the near future anyways.